packages/client/css/app.css

```diff
@@ -0,0 +1,106 @@
+/* Intended for use anywhere, more utilities than components */
+@import 'generic/page.css';
+@import 'generic/input.css';
+@import 'generic/button.css';
+
+@import 'generic/message-editor.css';
+@import 'generic/message-group.css';
+@import 'generic/message.css';
+
+/* One-time use or intended for use in a specific position in the document */
+@import 'specific/sidebar.css';
+@import 'specific/user-list.css';
+@import 'specific/modal.css';
+@import 'specific/message-list.css';
+
+/* Pages */
+@import 'pages/emote-settings.css';
+@import 'pages/account-settings.css';
+@import 'pages/authorized-users.css';
+
+body {
+  margin: 0;
+  padding: 0;
+
+  font-size: 16px;
+  font-family: system-ui, -apple-system, 'Segoe UI', Roboto, Oxygen-Sans, Ubuntu, Cantarell, Lato, 'Helvetica Neue', sans-serif;
+
+  --gray-900: #fff;
+  --gray-700: #eaedf2;
+  --gray-500: #cdd1d7;
+  --gray-300: #a3a8af;
+  --gray-100: #2d323a;
+
+  --blue: #337df4;
+  --blue-a2: rgba(51, 125, 244, 0.2); /* --blue with alpha of 0.2 */
+  --blue-a3: rgba(51, 125, 244, 0.3); /* --blue with alpha of 0.3 */
+
+  --green: #26a527;
+  --green-a2: rgba(38, 165, 39, 0.2); /* --green with alpha of 0.2 */
+  --green-a3: rgba(38, 165, 39, 0.3); /* --green with alpha of 0.3 */
+
+  --red: #f43259;
+}
+
+#app {
+  display: flex;
+  flex-direction: row;
+  align-items: stretch;
+
+  color: var(--gray-100);
+  background: var(--gray-900);
+
+  & > main {
+    display: flex;
+    flex-direction: column;
+
+    flex: 12 0 0;
+    height: 100vh;
+    overflow-y: hidden;
+
+    background: var(--gray-700);
+  }
+}
+
+@keyframes wiggle {
+  0%, 50%, 100% {
+    transform: translateZ(0);
+  }
+
+  15%, 23%, 31% {
+    transform: scale(1.03) rotate(-4deg) translateZ(0);
+  }
+
+  19%, 27%, 35% {
+    transform: scale(1.03) rotate(4deg) translateZ(0);
+  }
+}
+
+/* TODO: apply this based on state of elements (.is-empty), not as its own class */
+.wiggle {
+  animation: 3s infinite normal wiggle;
+}
+
+a:any-link.link, .Link {
+  color: var(--blue);
+  text-decoration: underline;
+}
+
+::-moz-selection { background: #b3d4fc; }
+::selection { background: #b3d4fc; }
+
+table, th, td {
```

Comment:
Feeling a little nervous about such generic selectors. Which tables are these styles meant to apply to? Where do they appear?

Reply:
The only tables in use are currently in the emote and authorized users setting pages, I think.

Reply:
Yeah, these are leftovers! We should move them to more specific places.